src/reference/02-DetailTopics/01-Using-sbt/06-Scripts.md

```diff
@@ -30,7 +30,7 @@ mode that only requires a JRE installed.
 Install [conscript](https://github.com/n8han/conscript).
 
 ```
-\$ cs sbt/sbt --branch 0.12.0
+\$ cs sbt/sbt --branch 0.13.0
```

Comment:
This is using `0.13.0` branch instead of the latest `0.13`.
I think the best way is to say `$app_version$` there, which the documentation engine would substitute to 0.13.7 from https://github.com/sbt/website/blob/master/src/reference/template.properties